lib/rubocop/cop/layout/empty_lines_around_arguments.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Layout
+      # This cops checks if empty lines exist around the arguments
+      # of a method invocation.
+      #
+      # @example
+      #   # bad
+      #   do_something(
```

Comment:
I guess you can a few more of the examples here - namely something with more arguments.